testsuites/CBLTester/CBL_Functional_tests/TestSetup_FunctionalTests/test_transactions.py

```diff
@@ -48,7 +49,10 @@ def test_transactions_insert_replace_remove_rollback(params_from_base_test_setup
     cluster_servers = topology["couchbase_servers"]
     cbs_one_url = cluster_servers[0]
     cb_server = couchbaseserver.CouchbaseServer(cbs_one_url)
+    server_version = get_cbs_version(cluster_config)
 
+    if server_version < "6.6.1":
+        pytest.skip("Cannot support server below 6.6.1 ")
```

Comment:
move the skip to top of the test, this way we avoid lot of nationalizations and small methods calls.  There is another skip call in the test, may be you can move it there

Reply:
done